### ISSUE TITLE
Bug: Reboot is required to make music appear in the default Music app

### ISSUE THREAD

**Issue by likeadragonmaid** [ubports/ubuntu-touch#2042]
- Device: Mi A2
- Channel: devel
- Build: 2022-09-05

### Steps to reproduce

1. Copy music files to ~/Music
2. Launch music app
3. Notice that the newly copied music files does not show up in Music app
4. Reboot device
5. Launch Music app
6. Notice that the new music files now show up in Music app

### Expected behavior
After adding new music to Music directory in home, music should appear in Music app

### Actual behavior
After adding new music to Music directory in home, music does not appear in Music app unless device is rebooted

**Comment by lduboeuf** [comment]
Just tried by copying from  local PC to Music directory by using a File Manager, music file is well recognized and shown in Music App, the same if using UT fileManager from Documents to Music.

You should get some informations if music files are well recognized here: ~/.cache/upstart/mediascanner-2.0.log or in the music app logs

**Comment by likeadragonmaid** [comment]
1. I moved a music file from ~/Music to ~/Downloads
2. Deleted all the files under ~/Music
3. Rebooted the device
4. Moved the music file from ~/Downloads to ~/Music
5. Launched the Music app
6. No Music detected
7. Copy log
8. Reboot device again
9. Music file appears
[step-7.zip](https://github.com/ubports/ubuntu-touch/files/9496157/step-7.zip)

**Comment by lduboeuf** [comment]
@gibcheesepuffs  according to your logs, MediaScanner doesn't seems to do anything.
Is it a known bug for that device ?

**Comment by likeadragonmaid** [comment]
No, I noticed it for the first time.

**Comment by Flohack74** [comment]
How many files are there? It might be just very slow if they pop in or idk.

**Comment by likeadragonmaid** [comment]
I tried with 1 file and 30+ files resulting in same behavior.

**Comment by lduboeuf** [comment]
@gibcheesepuffs how did you moved files ?, via FileManager in UT, via command line or with a PC ?

**Comment by likeadragonmaid** [comment]
Terminal app

**Comment by lduboeuf** [comment]
i see, but that should also work. Maybe you are using `mv` command ?

**Comment by likeadragonmaid** [comment]
You are correct, I used mv command.

**Comment by lduboeuf** [comment]
just try with `cp` then :). So there is still an issue with media scanner. But that use case is so rare...

**Comment by likeadragonmaid** [comment]
Yes, using cp command makes music show up in Music app without a reboot.
